streams/src/test/resources/log4j.properties

```diff
@@ -26,6 +26,7 @@ log4j.logger.org.apache.kafka.clients=ERROR
 # These are the only logs we will likely ever find anything useful in to debug Streams test failures
 log4j.logger.org.apache.kafka.clients.consumer=INFO
 log4j.logger.org.apache.kafka.clients.producer=INFO
+log4j.logger.org.apache.kafka.common.telemetry.internals.ClientTelemetryUtils=DEBUG
```

Comment:
That doesn't belong in this PR, does it?

Reply:
Yeah - I don't know where that came from